4-storage/relational-database-shredder/project/assembly.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
```

Comment:
This file breaks build:

```
4-storage/relational-database-shredder:r89 ❯ sbt                                                                                                                        ✱ ◼ 17:04:09
[info] Loading global plugins from /Users/chuwy/.sbt/0.13/plugins
[info] Loading project definition from /Users/chuwy/workspace/snowplow/4-storage/relational-database-shredder/project
[info] Updating {file:/Users/chuwy/workspace/snowplow/4-storage/relational-database-shredder/project/}relational-database-shredder-build...
[warn] Multiple dependencies with the same organization/name but different versions. To avoid conflict, pick one version:
[warn]  * com.eed3si9n:sbt-assembly:(0.14.3, 0.14.4)
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[warn] There may be incompatibilities among your library dependencies.
[warn] Here are some of the libraries that were evicted:
[warn]  * com.eed3si9n:sbt-assembly:0.14.3 -> 0.14.4
[warn] Run 'evicted' to see detailed eviction warnings
[info] Compiling 2 Scala sources to /Users/chuwy/workspace/snowplow/4-storage/relational-database-shredder/project/target/scala-2.10/sbt-0.13/classes...
[error] /Users/chuwy/workspace/snowplow/4-storage/relational-database-shredder/project/BuildSettings.scala:49: not found: value sbtassembly
[error]   import sbtassembly.AssemblyPlugin.autoImport._
[error]          ^
[error] /Users/chuwy/workspace/snowplow/4-storage/relational-database-shredder/project/BuildSettings.scala:52: not found: value assemblyJarName
[error]     assemblyJarName in assembly := { name.value + "-" + version.value + ".jar" },
[error]     ^
[error] /Users/chuwy/workspace/snowplow/4-storage/relational-database-shredder/project/BuildSettings.scala:55: not found: value assemblyExcludedJars
[error]     assemblyExcludedJars in assembly := {
[error]     ^
[error] /Users/chuwy/workspace/snowplow/4-storage/relational-database-shredder/project/BuildSettings.scala:71: not found: value assembly
[error]     test in assembly := {},
[error]             ^
[error] /Users/chuwy/workspace/snowplow/4-storage/relational-database-shredder/project/BuildSettings.scala:72: not found: value assemblyMergeStrategy
[error]     assemblyMergeStrategy in assembly := {
[error]     ^
[error] 5 errors found
[error] (compile:compileIncremental) Compilation failed
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? q
```